Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -0,0 +1,598 @@
+      throw new NotImplementedException();
+    }
+
+    protected override void EmitConversionExpr(ConversionExpr e, bool inLetExprBody, ConcreteSyntaxTree wr) {
+      throw new NotImplementedException();
+    }
+
+    protected override void EmitTypeTest(string localName, Type fromType, Type toType, IToken tok,
+      ConcreteSyntaxTree wr) {
+      throw new NotImplementedException();
+    }
+
+    protected override void EmitCollectionDisplay(CollectionType ct, IToken tok, List<Expression> elements,
+      bool inLetExprBody, ConcreteSyntaxTree wr) {
+      throw new NotImplementedException();
+    }
+
+    protected override void EmitMapDisplay(MapType mt, IToken tok, List<ExpressionPair> elements, bool inLetExprBody,
+      ConcreteSyntaxTree wr) {
+      throw new NotImplementedException();
+    }
+
+    protected override void EmitSetBuilder_New(ConcreteSyntaxTree wr, SetComprehension e, string collectionName) {
+      throw new NotImplementedException();
+    }
+
+    protected override void EmitMapBuilder_New(ConcreteSyntaxTree wr, MapComprehension e, string collectionName) {
+      throw new NotImplementedException();
+    }
+
+    protected override void EmitSetBuilder_Add(CollectionType ct, string collName, Expression elmt, bool inLetExprBody,
+      ConcreteSyntaxTree wr) {
+      throw new NotImplementedException();
+    }
+
+    protected override ConcreteSyntaxTree EmitMapBuilder_Add(MapType mt, IToken tok, string collName, Expression term,
+      bool inLetExprBody,
+      ConcreteSyntaxTree wr) {
+      throw new NotImplementedException();
+    }
+
+    protected override string GetCollectionBuilder_Build(CollectionType ct, IToken tok, string collName,
+      ConcreteSyntaxTree wr) {
+      throw new NotImplementedException();
+    }
+
+    protected override void EmitSingleValueGenerator(Expression e, bool inLetExprBody, string type,
+      ConcreteSyntaxTree wr) {
+      throw new NotImplementedException();
+    }
+
+    public override bool TextualTargetIsExecutable => true;
+
+    public override bool CompileTargetProgram(string dafnyProgramName, string targetProgramText,
+      string /*?*/ callToMain, string /*?*/ targetFilename, ReadOnlyCollection<string> otherFileNames,
+      bool runAfterCompile, TextWriter outputWriter, out object compilationResult) {
+      compilationResult = null;
+      if (runAfterCompile) {
+        Contract.Assert(callToMain != null); // this is part of the contract of CompileTargetProgram
+        // Since the program is to be run soon, nothing further is done here. Any compilation errors (that is, any errors
+        // in the emitted program--this should never happen if the compiler itself is correct) will be reported as 'node'
+        // will run the program.
+        return true;
+      } else {
+        // compile now
+        return SendToNewNodeProcess(dafnyProgramName, targetProgramText, null, targetFilename, otherFileNames,
```

Comment:
```suggestion
        return SendToNewPythonProcess(dafnyProgramName, targetProgramText, null, targetFilename, otherFileNames,
```

And likewise below.

Reply:
Oh yes, I changed this before but forgot to do it again after pulling in previous commit due to errors.